src/nerve/frontends/tui/commander/blocks.py

```
@@ -0,0 +1,267 @@
+    error: str | None = None
+    raw: dict[str, Any] = field(default_factory=dict)  # Raw result from node
+
+    # Metadata
+    number: int = 0  # Block number (set by Timeline)
+    duration_ms: float | None = None
+    metadata: dict[str, Any] = field(default_factory=dict)
+
+    # Execution status: "pending", "running", "completed", "error"
+    status: str = "completed"
+
+    def render(self, console: Console, show_separator: bool = True) -> RenderableType:
```

Ruff (0.14.10)

Unused method argument: `console`

(ARG002)